core/src/main/java/com/linecorp/armeria/server/logging/LoggingService.java

```diff
@@ -35,20 +38,22 @@
  */
 public final class LoggingService extends SimpleDecoratingHttpService {
 
+    private static final Logger logger = LoggerFactory.getLogger(LoggingService.class);
+
     /**
      * Returns a new {@link HttpService} decorator that logs {@link HttpRequest}s and {@link HttpResponse}s at
      * {@link LogLevel#DEBUG} for success, {@link LogLevel#WARN} for failure. See {@link LoggingServiceBuilder}
      * for more information on the default settings.
      */
     public static Function<? super HttpService, LoggingService> newDecorator() {
-        return builder().newDecorator();
+        return new LoggingServiceBuilder().defaultLogger(logger).newDecorator();
```

Comment:
```suggestion
        return builder().newDecorator();
```

Reply:
You mean `builder().defaultLogger(logger).newDecorator();`? I have to set the default logger. 😉

Reply:
Never mind. 😓